### PR TITLE
Feature/smartbch connect any bch node

### LINKED ISSUE #44: Support remote BCH RPC
Currently the devsuite need local BCH RPC deployed via devsuite script, it should be configurable to connect to any remote BCH node.

### PR DESCRIPTION
**Pull request by tleekkul**
Closes #44

### CODE REVIEW

File: bch-devsuite [hunk 1]
```diff
@@ -200,6 +200,17 @@ def handle_smartbch():
     elif config.network in (Network.TESTNET, Network.MAINNET):
         config.smartbch = SmartBCHConf(enabled=True)
 
+def handle_bch_rpc_conf():
+    host = inquirer.text(message="BCH RPC host", default="http://bch-node")
```
Comment (SuperCipher) [inline bch-devsuite:204]: Where is this node? `http://bch-node`
message should be `"BCH RPC host *WITH PROTOCOL*"`

Comment (SuperCipher) [reply bch-devsuite:204]: Why not default to local node?

File: bch-devsuite [hunk 2]
```diff
@@ -200,6 +200,17 @@ def handle_smartbch():
...
+    port = inquirer.text(message="BCH RPC port", default=config.rpc_port())
+    username = inquirer.text(message="BCH RPC username", default="actorforth")
```
Comment (SuperCipher) [inline bch-devsuite:206]: Can we have it random by default?
We have encountered people messing around to the node with the default password multiple times.
This is a massive complaint I saw in other BCH tools.

File: bch-devsuite (hunk 1 above)
Comment (tleekkul) [reply bch-devsuite:204]: It's the local node, it's hostname inside docker-compose network

File: bch-devsuite (hunk 2 above)
Comment (tleekkul) [reply bch-devsuite:209]: Updated

**Review by SuperCipher — APPROVED** [review]

**Pull request merged**